sdk/identity/azure-identity/src/test/resources/key.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
This key is generated as a test resource from https://csfieldguide.org.nz/en/interactives/rsa-key-generator/

And it is not the private key of the above certificate.